website/common/script/ops/buy/buyQuest.js

```diff
@@ -10,7 +10,7 @@ import get from 'lodash/get';
 // buy a quest with gold
 module.exports = function buyQuest (user, req = {}, analytics) {
   let key = get(req, 'params.key');
-  let quantity = req.quantity || 1;
+  let quantity = Number(req.quantity) || 1;
```

Comment:
this works but I feel like we should also throw an error if the param couldn't be converted to a number correctly, so add another check after this like `Number.isInteger(quantity)`and if it's false, throw an error